netbout-client/src/main/java/com/netbout/mock/H2Sql.java

```diff
@@ -48,6 +48,7 @@
 @ToString
 @Loggable(Loggable.DEBUG)
 @EqualsAndHashCode(of = "file")
+@SuppressWarnings("PMD.ConstructorOnlyInitializesOrCallOtherConstructors")
```

Comment:
@mkordas There is a commit which updated netbout/pom.xml with qulice version 0.16.2.